secondlargest.js

```diff
@@ -0,0 +1,17 @@
+// Problem 1: Complete the secondLargest function which takes in an array of numbers in input and return the second biggest number in the array. (without using sort)?
+function secondLargest(array) {
+  var largest =0, secondLargest = 0, length;
```

Comment:
1. Be consistent in your formatting **ALWAYS**! The equals sign has a space after it in one instance and not in one.
2. Follow the guidelines from the link provided on your trello card for whether there should be a space or not, and for all other such instances.